src/python/system/environment.py

```diff
@@ -471,6 +471,8 @@ def get_platform_id():
   """Return a platform id as a lowercase string."""
   bot_platform = platform()
   if is_android(bot_platform):
+    if is_android_emulator():
```

Comment:
move it to start, why put it inside "if is_android" ?

Reply:
still needs fixing.

Reply:
It should be in the latest commit (10).